### PR TITLE
Remove dockershim related files

### PR DESCRIPTION
**Pull request by jsturtevant**
Signed-off-by: James Sturtevant <jstur@microsoft.com>

**Reason for PR**:

Removes files that no longer work now that dockershim has been removed.  Now that 1.23 is the latest version of Kubernetes supported we can use the HostProcess containers solution.  

Files prior to this can be found at https://github.com/kubernetes-sigs/sig-windows-tools/releases/tag/v0.1.6

**Issue Fixed**:

Issue #

**Requirements**

- [ ] Sqaush commits 
- [x] Documentation
- [ ] Tests

**Notes**:

/assign @marosset 
/hold

### CODE REVIEW

File: kubeadm/scripts/Install-Containerd.ps1 [hunk 1]
```diff
@@ -1,90 +0,0 @@
-<#
-.SYNOPSIS
-Installs ContainerD on a Windows machines in preperation for joining the node to a Kubernetes cluster.
```
Comment (fabi200123) [inline kubeadm/scripts/Install-Containerd.ps1:3]: Should we not move this script as well? I believe this is still needed to install ContainerD on Windows

Comment (jsturtevant) [reply kubeadm/scripts/Install-Containerd.ps1:3]: yes, good catch!

**Comment by jsturtevant** [comment]
fyi @Mik4sa

File: kubeadm/scripts/Install-Containerd.ps1 (hunk 1 above)
Comment (jsturtevant) [reply kubeadm/scripts/Install-Containerd.ps1]: fixed

**Review by marosset — APPROVED** [review]
/lgtm

**Pull request merged**